content/applications/marketing/email_marketing/analyze_metrics.rst

```diff
@@ -0,0 +1,111 @@
+===============
+Analyze metrics
+===============
+
+.. |CTR| replace:: :abbr:`CTR (Click through rate)`
+
+To properly understand the success or failure of an email marketing campaign, it is
+necessary to monitor several key metrics. The insights gained from these metrics can then be used to
+optimize future campaigns. Odoo's **Email Marketing** application tracks several :ref:`key metrics
+<email-marketing/key-metrics>`, that can be interpreted to improve future campaigns.
+
+View metrics
+============
+
+After a mass mail has been sent, the results for that particular mailing are displayed in multiple
+locations.
+
+To access the metrics for an individual mailing, navigate to :menuselection:`Email Marketing app
+--> Mailings`. Locate the specific mailing in the list view, and use the column headings to view the
+results for that mailing. Click on one of the mailings in the list to open the record. At the top of
+the record, detailed metrics are displayed as smart buttons.
+
+.. image:: analyze_metrics/metric-smart-buttons.png
+   :align: center
+   :alt: The smart buttons on a mass mailing, displaying the results of the message.
+
+.. tip::
+   Click on the smart buttons to see the recipient records for each metric.
+
+.. _email-marketing/key-metrics:
+
+Opened rate
+-----------
+
+The *Opened* rate measures the percentage of emails opened by recipients against the total number of
+sent emails.
+
+A high open rate may indicate that the subject line was timely, compelling, and successfully
+prompted the recipients to view the message.
+
+A low open rate may indicate the subject line failed to capture the recipients' interest. It could
+also indicate the email ended up in a spam or junk folder due to having a poor sender reputation or
+failing to :doc:`configure the proper DNS records <../../general/email_communication/email_domain>`.
+
+Replied rate
+------------
+
+The *Replied* rate measures the percentage of recipients who responded to the email directly against
+the total number of sent emails.
+
+A high replied rate may indicate the email resonated with recipients, prompting them to take action
+or provide feedback.
+
+A low replied rate may suggest that the message lacked relevance or did not contain a clear call to
+action.
+
+Clicked rate
+------------
+
+The *Clicked* through rate (CTR), measures the percentage of recipients who clicked on a link within
```

Comment:
This may be an instance wherein the GUI label of the button might be more useful.

For instance, this could be reworded to:
"The :guilabel:`Clicked` smart button represents the *Click Through Rate (CTR)*, which measures the percentage..."

Or something like that...
```suggestion
The *Clicked* rate, represents the *Click Through Rate (CTR)*, which measures the percentage of recipients who clicked on a link within
```